port/common/omrmem32helpers.c

```diff
@@ -71,7 +60,9 @@ startup_memory32(struct OMRPortLibrary *portLibrary)
 	PPG_mem_mem32_subAllocHeapMem32.subCommitCommittedMemorySize = 0;
 	PPG_mem_mem32_subAllocHeapMem32.subCommitHeapWrapper = NULL;
 	PPG_mem_mem32_subAllocHeapMem32.suballocator_initialSize = 0;
-	PPG_mem_mem32_subAllocHeapMem32.suballocator_commitSize = 0;
+	PPG_mem_mem32_subAllocHeapMem32.suballocator_commitSize = SUBALLOCATOR_COMMIT_SIZE;
+	PPG_mem_mem32_subAllocHeapMem32.suballocator_incrementSize = SUBALLOCATOR_INCREMENT_SIZE;
+	PPG_mem_mem32_subAllocHeapMem32.suballocator_quickAlloc = TRUE;
```

Comment:
iirc, the quick alloc optimization is only available on Linux platforms. Will these default values work on all platforms?

Reply:
For documentation, we should also add a note about its platform specific nature near its declaration (e.g. `Linux only`)